frontend/src/models/groupsModel.ts

```diff
@@ -26,8 +26,8 @@ export const groupsModel = kea<groupsModelType>({
     }),
     selectors: {
         showGroupsOptions: [
-            (s) => [s.groupsEnabled, s.groupTypes],
-            (enabled, groupTypes) => enabled && groupTypes.length > 0,
+            (s) => [s.groupsAccessStatus, s.groupsEnabled, s.groupTypes],
+            (status, enabled, groupTypes) => status !== GroupsAccessStatus.Hidden || (enabled && groupTypes.length > 0),
```

Comment:
Doesn't the `groupsAccessStatus` already check whether it's enabled and/or has added a group? Perhaps just check that status is not `Hidden` or `AlreadyUsing` ?

Reply:
This would cause the issue you outline below where we don't show the dropdowns.